cms/djangoapps/contentstore/views/block.py

```diff
@@ -74,6 +75,7 @@
 ALWAYS = lambda x: True
 
 
+@transaction.non_atomic_requests
```

Comment:
Please add comments above this indicating why this is important to have here, particularly for the "do stuff after publish" use case.

Reply:
Done! Thank you